lib/public/views/Runs/Runs.js

```diff
@@ -196,13 +196,47 @@ export default class Overview extends Observable {
 
     /**
      * Filter runs based on the selected run numbers and fields to prepare it for export
```

Comment:
One last small change.
Would you mind updating the method documentation with the new functionality?